### PR TITLE
36652- I2-static-page

### PR DESCRIPTION
**Pull request by vinhnd2010**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,7 +11,7 @@ gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.1.0'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
-
+gem "mysql2"
```
Comment (ThuBM) [inline Gemfile]: remove

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    }
+  }
+
+  a {
+    color: #555;
+    &:hover {
+      color: #222;
+    }
+  }
+
```
Comment (ThuBM) [inline app/assets/stylesheets/custom.css.scss]: remove

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t("static_page.logo"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t("user.index"),   " " %></li>
```
Comment (ThuBM) [inline app/views/layouts/_header.html.erb]: coding convention

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        <% if %>
```
Comment (ThuBM) [inline app/views/layouts/_header.html.erb]: if what?????

File: app/assets/javascripts/static_pages.js.coffee [hunk 5]
```diff
@@ -0,0 +1,3 @@
+$(document).ready ->
+  $("div.alert").delay(3000).slideUp()
+  return
```
Comment (ThuBM) [inline app/assets/javascripts/static_pages.js.coffee]: remove

File: app/views/layouts/_footer.html.erb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<footer class = "footer">
+  <small>
+    <a href=""><%= t "static_page.brs" %></a>
+    - <a href=""><%= t "static_page.trainteam" %></a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to t("static_page.about"), "" %></li>
```
Comment (ThuBM) [inline app/views/layouts/_footer.html.erb]: need about link ????? no have about method

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -1,14 +1,25 @@
...
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
+  <%= render "layouts/header" %>
 </head>
 <body>
-
-<%= yield %>
-
+  <div class="container">
+    <% flash.each do |message_type, message| %>
```
Comment (ThuBM) [inline app/views/layouts/application.html.erb]: write in _flash.html.erb

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -1,14 +1,25 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Base</title>
+  <title><%= full_title yield :title %></title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
```
Comment (ThuBM) [inline app/views/layouts/application.html.erb]: coding convention:
'  => ""
"data-turbolinks-track": true

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -1,14 +1,25 @@
...
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
```
Comment (ThuBM) [inline app/views/layouts/application.html.erb]: như trên

File: app/views/layouts/application.html.erb [hunk 10]
```diff
@@ -1,14 +1,25 @@
...
   <%= csrf_meta_tags %>
+  <%= render "layouts/header" %>
```
Comment (ThuBM) [inline app/views/layouts/application.html.erb]: move to body

File: app/views/layouts/_header.html.erb [hunk 11]
```diff
@@ -0,0 +1,23 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t("static_page.logo"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t("user.index"), " " %></li>
+        <li class="dropdown">
+          <a href="" class= "dropdown-toggle" data-toggle="dropdown">
+            <%= t("user.account") %><b class = "caret"></b>
```
Comment (ThuBM) [inline app/views/layouts/_header.html.erb]: class="caret"

File: app/views/layouts/application.html.erb [hunk 12]
```diff
@@ -1,14 +1,18 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Base</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <title><%= full_title yield :title %></title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': true %>
```
Comment (ThuBM) [inline app/views/layouts/application.html.erb]: remove ' => "
3 space after style_sheet_link_tag

File: app/views/layouts/application.html.erb [hunk 13]
```diff
@@ -1,14 +1,18 @@
...
+  <%= javascript_include_tag 'application', 'data-turbolinks-track': true %>
```
Comment (ThuBM) [inline app/views/layouts/application.html.erb]: ' => "

**Comment by ThuBM** [comment]
ok

File: Gemfile.lock [hunk 14]
```diff
@@ -42,8 +42,11 @@ GEM
       debug_inspector (>= 0.0.1)
     bootstrap-sass (3.2.0.2)
       sass (~> 3.2)
+<<<<<<< HEAD
```
Comment (chuanhtuan) [inline Gemfile.lock]: ???

**Pull request merged**